src/main/java/com/iota/iri/service/snapshot/impl/LocalSnapshotManagerImpl.java

```diff
@@ -51,6 +57,11 @@
      * Configuration with important snapshot related parameters.
      */
     private SnapshotConfig config;
+    
+    /**
+     * If this node is currently seen as in sync
+     */
+    private boolean isInSync;
```

Comment:
I don't like this field:
1. It just states that this class takes more memory
2. You can easily do without it and I don't understand what value it adds
3. For this class it isn't a good example, but in general in multi threaded environments you don't want to have these kind of things since 2 different threads may modify this field.